cpu/sam0_common/periph/spi.c

```diff
@@ -102,7 +102,8 @@ void spi_init(spi_t bus)
 
 void spi_init_pins(spi_t bus)
 {
-    gpio_init(spi_config[bus].miso_pin, GPIO_IN);
+    //MISO must always have PD/PU, see #5968. This is a ~65uA difference
```

Comment:
Ok, fixed. I changed this comment style and combined the commits for cleaner merging.